kubernetes/helm_charts/sunbird-RC/registry/templates/credential_template.json

```diff
@@ -1,7 +1,7 @@
 {
     "@context": [
-    "https://raw.githubusercontent.com/project-sunbird/sunbird-devops/release-4.8.0/kubernetes/helm_charts/sunbird-RC/registry/templates/v1_context.json",			
-    "https://raw.githubusercontent.com/project-sunbird/sunbird-devops/release-4.8.0/kubernetes/helm_charts/sunbird-RC/registry/templates/sunbird_context.json"
+    "https://gist.githubusercontent.com/reshmi-nair/386ef6d832236014b7d1037ad4de3da7/raw/8288d9739f3e7946740b4119d0f5f1991695c2a5/v1_context.json",			
```

Comment:
No links from personal repos allowed.

Reply:
currently, to unblock the  testing purpose we have used the url.. we will be moving to azure these files and then we will change it public repo.. 
@reshmi-nair